test/vc-data-model-1.0/50-jwt.js

```diff
@@ -72,11 +72,11 @@ describe('JWT (optional)', () => {
         const jwtResult = new jwt.JwsToken(jwtBase64);
         expect(jwtResult.isContentWellFormedToken()).to.be.true;
 
-        const alg = jwtResult.getHeader('alg')
+        const alg = jwtResult.getHeader()['alg']
         expect(alg).to.be.a('string')
 
         let publicKey;
-        if (jwtResult.getHeader('alg') === 'ES256K') {
+        if (jwtResult.getHeader()['alg'] === 'ES256K') {
```

Comment:
we can reuse the previously assigned variable `alg` here instead of calling the function again